driver/others/dynamic.c

```diff
@@ -338,6 +460,23 @@ static gotoblas_t *get_coretype(void){
 	  return &gotoblas_NEHALEM;
 	}	
 	return NULL;
+      case 6:
```

Comment:
this should go in independent of this PR no?

Reply:
That is why I mentioned the local tree being out of sync and this PR being just for discussion.

Reply:
ah nm me then.